src/call/github-workflow-runs-monitor.ts

```diff
@@ -70,4 +81,35 @@ export default async function githubWorkflowRunsMonitor(app: Probot, context: an
   } catch (error) {
     app.log.error(`Error indexing log data: ${error}`);
   }
+
+  let count: any;
```

Comment:
What count? should be number or any here.

Reply:
The count (0 or 1) that will be added to the cloudwatch alarm

Reply:
> The count (0 or 1) that will be added to the cloudwatch alarm

Can we use a boolean instead?

Reply:
We cant use boolean as cloudwatch count value expects 0 or 1 peter.